eng/docker/rhel.Dockerfile

```
@@ -1,5 +1,5 @@
 # Dockerfile that creates a container suitable to build dotnet-cli
-FROM mcr.microsoft.com/dotnet-buildtools/prereqs:rhel-7-rpmpkg-e1b4a89-20175311035359
+FROM mcr.microsoft.com/dotnet-buildtools/prereqs:centos-7-rpmpkg-20210714125435-9b5bbc2
```

👍

---

```suggestion
FROM mcr.microsoft.com/dotnet-buildtools/prereqs:centos-8-rpmpkg-20210714125410-daa5116
```

From https://github.com/dotnet/versions/blob/a3164ca1c7f07c6ad990834bd741e16ac24b3ec5/build-info/docker/image-info.dotnet-dotnet-buildtools-prereqs-docker-main.json#L745